packages/modelviewer.dev/scripts/ci-before-deploy.sh

```diff
@@ -105,6 +105,12 @@ mv ../render-fidelity-tools/test/results $DEPLOY_ROOT/fidelity/results
 cp ../render-fidelity-tools/test/results-viewer.html $DEPLOY_ROOT/fidelity/index.html
 cp ../render-fidelity-tools/dist/* $DEPLOY_ROOT/dist/
 
+FILES_TO_PATCH_WITH_MINIFIED_BUNDLE=($(find $DEPLOY_ROOT \( -type d -name node_modules -prune \) -o -type f | grep \.html))
+
+for file_to_patch in "${FILES_TO_PATCH_WITH_MINIFIED_BUNDLE[@]}"; do
+  sed -i '' 's/model-viewer\.js/model-viewer\.min\.js/g' $file_to_patch
```

Comment:
Is this replacing non-minified with minified? Don't the pages have the minified version now?